extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -103,13 +104,14 @@ export class AmpSidebar extends AMP.BaseElement {
       this.element.setAttribute('side', this.side_);
     }
 
+    const ampDoc = this.getAmpDoc();
     // Get the toolbar attribute from the child navs
     const toolbarElements =
-    Array.prototype.slice
-      .call(this.element.querySelectorAll('nav[toolbar]'), 0);
+      toArray(this.element.querySelectorAll('nav[toolbar][toolbar-target]'));
```

Comment:
Let instead search for `nav[toolbar]`, but `user().assert()` that it has `toolbar-item`.